setup.py

```diff
@@ -25,8 +25,7 @@
 exec(open(os.path.join(".", "dwave", "system", "package_info.py")).read())
 
 
-# if the dimod version range changes, update in .circleci/config.yml
-install_requires = ['dimod>=0.9.11,<0.10.0',
+install_requires = ['dimod>=0.9.11,<=0.10.0.dev7',
```

Comment:
does the lower bound here need to be updated to 0.9.15?

Reply:
We test `0.9.11` in the CI, so I think we're OK. I picked 0.9.15 somewhat arbitrarily